nixos/modules/services/web-servers/tomcat.nix

```diff
@@ -23,7 +23,7 @@ in
 
       package = mkOption {
         type = types.package;
-        default = pkgs.tomcat85;
+        default = pkgs.tomcat9;
         defaultText = literalExpression "pkgs.tomcat85";
```

Comment:
this line also has to be changed to match

Reply:
Ah yes, good catch. I just did a grep on anything referring to `tomcat8` and found another bad example. I've just corrected it and amended the commit.